src/main/java/haveric/recipeManager/Vanilla.java

```diff
@@ -142,7 +142,7 @@ protected static void init() {
             initialRecipes.put(new FuelRecipe(Material.FISHING_ROD, 15), info);
             initialRecipes.put(new FuelRecipe(Material.SIGN, 10), info);
             initialRecipes.put(new FuelRecipe(Material.BOWL, 5), info);
-            initialRecipes.put(new FuelRecipe(Material.WOODEN_DOOR, 10), info);
+            initialRecipes.put(new FuelRecipe(Material.WOOD_DOOR, 10), info); // was WOODEN_DOOR, which is block type, not item type...

```

Comment:
It looks to have been added here: https://github.com/haveric/RecipeManager2/commit/c0ad54615d281eb8b45802620bcaaca6413c22c7 as part of a larger 1.11 push. I don't see other references to it casually, without digging into prior MC version NMS's.